packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -9,26 +9,29 @@ relatedComponents: ['task-list']
 
 ```jsx live
 <ProgressIndicator
+	activePath="#organisations"
 	items={[
-		{ label: 'Introduction', status: 'done', href: '#' },
-		{ label: 'Submit evidence', status: 'saved', href: '#' },
-		{ label: 'Organisations', status: 'started', href: '#' },
-		{ label: 'Business contacts', status: 'error', href: '#' },
-		{ label: 'Case studies', status: 'todo', href: '#' },
-		{ label: 'Attachments', status: 'started', href: '#', isActive: true },
-		{ label: 'Review and submit', status: 'blocked', href: '#' },
+		{ label: 'Introduction', status: 'done', href: '#introduction' },
+		{ label: 'Submit evidence', status: 'saved', href: '#evidence' },
+		{ label: 'Organisations', status: 'started', href: '#organisations' },
+		{ label: 'Business contacts', status: 'error', href: '#contacts' },
+		{ label: 'Case studies', status: 'todo', href: '#case-studies' },
+		{ label: 'Attachments', status: 'started', href: '#attachments' },
+		{ label: 'Review and submit', status: 'blocked', href: '#review' },
 	]}
 />
 ```
 
-Progress indicators are used to show users how many steps are needed to complete a form and where they are in the process of completing it. This helps users know how much is left to do and how far along they are.
+Progress indicators are used to help users navigate to each step of a multi-step form, show users how many steps are needed to complete and where they are in the process of completing it. This helps users know how much is left to do and how far along they are.
 
 Progress indicators can be applied to larger tasks that can be completed in any order and returned to at a later date, like doing a tax return or applying for a driver’s licence.
 
 Progress indicators can be applied to multi-step forms if the form has a known number of steps to be completed. <sup><a href="#cite-1">1</a></sup>
 
 Avoid adding or removing pages to a multi-step form where a progress indicator is used as this can confuse and mislead users. Consider another approach if the number of steps might change due to user input. <sup><a href="#cite-2">2</a></sup>
 
+Progress indicators contain level 1 links to each page of a multi-step form. When additional pages are required for secondary actions nested in level 1 pages, level 2 links are revealed in the Progress indicator to help a user understand where they have navigated to and how to return to the to the previous step.
```

Comment:
This feels confusing, verbose and has some typos. How about something like this:

```suggestion
By default, A Progress indicator contains links to each level 1 page of a multi-step flow.

When a user navigates to a secondary action page from a level 1 page, a nested level 2 link is revealed in the Progress indicator to help a user understand where they have navigated to and how to return to the previous step.
```

Reply:
Should the first sentence be `By default, a`? i.e. lowercase `a`?

Reply:
@adrian-yee what are your thoughts on @stowball 's copy suggestions?